css/index.css

```diff
@@ -0,0 +1,125 @@
+
+
+body 
+{background-color: #deb887;
+ text-align: left; /*is this defaut*/
+ font-family: "Oswald", sans-serif;
+ padding: 40px;}
+
+ *
+{margin:2%;}
+
+img 
+{width: 100%;}
+
+h1 
+{font-size: 30px;
+ font-weight: 700;
+ color: maroon;}
+
+h2 
+{font-size: 20px;
+ font-weight: 100;
+ color: maroon}
+
+#about, #liExp, #liProject
+{background-color: bisque;
+ margin: 10px;
+ border:3px solid;
+ border-radius: 10px;
+ padding: 20px;}
+ 
+ 
+
+
+#skills li 
+{background-color: orange;
+ margin: 10px;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 20px;}
+
+#liEmail
+{background-color: darkkhaki;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 10px;   
+ margin-bottom: -25px;
+ width: 134px;}
+
+#liGithub
+{background-color: darkkhaki;
+ border: 3px solid;
+ border-radius: 10px;
+ padding: 10px;
+ width: 165px;}
+
+
+#fL1 
+{font-size: 20px;
+ color: red;}
+
+#fL2 
+{font-size: 20px;
+ color: blue;}
+
+#fL3 
+{font-size: 20px;
+ color: green;}
+
+#fI1 
+{font-size: 20px;
+ color:  brown;
+ margin-bottom: 25px;}
+
+#fI2 
+{font-size: 20px;
+ color: brown;
+ margin-bottom: 25px;}
+
+#fI3 
+{font-size: 20px;
+ color: brown;
+ margin-bottom: 25px;}
```

Comment:
In these blocks you're applying `font-size: 20px` multiple times making your style sheet a bit bloated. This is a great use case for using classes rather than ID's. There are many ways to approach this, but here's a couple you might try:

1. Add `font-size` to the label element
2. Create a class specifically for setting font-size to 20px and re-use it each time you want your font to be that size
3. Nest a 'child' heading element with the appropriate size within the label tag.

Reply:
done